src/main/java/Duke/storage/Storage.java

```diff
@@ -0,0 +1,95 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+import duke.ui.DukeException;
+
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private String saveFilePath;
+
+    /**
+     * Constructor for Storage.
+     *
+     * @param saveFilePath the directory path for the save file.
+     */
+    public Storage(String saveFilePath) {
+        this.saveFilePath = saveFilePath;
+    }
+
+    /**
+     * Reads the existing file. If there is existing file, then add to the ArrayList.
+     *
+     * @return the the list of tasks.
+     * @throws DukeException If files cannot be read or created.
+     */
+    public ArrayList<Task> load() throws DukeException {
+        ArrayList<Task> taskList = new ArrayList<>();
+        try {
+            File saveFile = new File(saveFilePath);
+            saveFile.getParentFile().mkdir();
+            saveFile.createNewFile();
+            Scanner scanner = new Scanner(saveFile);
+            while (scanner.hasNext()) {
+                String currLine = scanner.nextLine();
+                String[] currTaskLine = currLine.split("\\|");
+                switch (currTaskLine[0]) {
+                case "T":
+                    ToDo todoTask = new ToDo(currTaskLine[2]);
```

Comment:
Is ToDo one-word or two-word? The class `ToDo` seems to be two-word, however the variable name `todoTask` seems to be one-word :)